Demo/Demo/ViewModels/PayPalViewModel.swift

```diff
@@ -104,11 +104,21 @@ extension PayPalViewModel: PayPalNativeCheckoutDelegate {
 
 extension PayPalViewModel: PayPalNativeShippingDelegate {
     
-    func paypal(_ payPalClient: PayPalNativeCheckoutClient, didShippingAddressChange shippingAddress: PayPalNativeShippingAddress) {
+    func paypal(
+        _ payPalClient: PayPalNativeCheckoutClient,
+        didShippingAddressChange shippingAddress: PayPalNativeShippingAddress,
```

Comment:
```suggestion
        didChangeShippingAddress shippingAddress: PayPalNativeShippingAddress,
```
🧶 to make it read like `paypal:didChangeShippingAddress:withAmount`. Got this from the [OG naming guidelines ("Delegate Methods" section)](https://developer.apple.com/library/archive/documentation/Cocoa/Conceptual/CodingGuidelines/Articles/NamingMethods.html).